profile/README.md

```diff
@@ -59,5 +59,5 @@ We recommend to choose a focus area first to get started:
 ## Community
 
 Looking to get involved? [Contributions are welcome](https://unikraft.org/docs/contributing)!
-There are [weekly communitiy meetings](https://unikraft.org/community/meetings) on [our Discord server](https://bit.ly/UnikraftDiscord) which follow [on-going projects](https://github.com/unikraft/unikraft/issues?q=is%3Aissue+is%3Aopen+label%3Alifecycle%2Factive).
+There are [weekly community events](https://unikraft.org/community/events) on [our Discord server](https://bit.ly/UnikraftDiscord) which follow [on-going projects](https://github.com/unikraft/unikraft/issues?q=is%3Aissue+is%3Aopen+label%3Alifecycle%2Factive).
```

Comment:
**thought**: Could be worth highlighting [hackathons](https://unikraft.org/hackathons) or linking to a parent community page.